app/policies/collection_draft_proposal_policy.rb

```diff
@@ -0,0 +1,29 @@
+class CollectionDraftProposalPolicy < DraftPolicy
```

Comment:
This works for when we were working on the first ticket, but I think we need to come up with a programmatic way of allowing or blocking access. I am thinking along the lines of using the environmental configuration and possibly checking that they are logged in through EDL, as those are the requirements to access proposals. `publish?` should probably just stay as `false` though.

Reply:
Now check for proposal mode and if user logged in